docs/Overview/index.md

```diff
@@ -48,3 +48,5 @@ Abbreviations used in this document are collected in the table below
 |VDOP|Vertical Dilution of Precision|
 |VR|Voice Recognition|
 |UTC|Universal Time Coordinate|
+|ASC|App Service Consumer|
+|ASP|App Service Provider|
```

Comment:
Can this be added as the second abbreviation, so the table is in alphabetical order?